src/data/constants/actionTypes.js

```diff
@@ -6,4 +6,5 @@ export const assetActions = {
   DELETE_ASSET_SUCCESS: 'DELETE_ASSET_SUCCESS',
   ASSET_XHR_FAILURE: 'ASSET_XHR_FAILURE',
   CLEAR_ASSETS_STATUS: 'CLEAR_ASSETS_STATUS',
+  SORT_UPDATE: 'SORT_UPDATE',
```

Comment:
Self explanatory, but we now have a new action type that signifies a change in the selected sort.